src/elements/pearl-chain/pearl-chain-leg/pearl-chain-leg.scss

```diff
@@ -0,0 +1,167 @@
+@use '../../core/styles' as sbb;
+
+@mixin sbb-pearl-chain-dotted {
+  background-color: unset;
+  background-image: linear-gradient(to right, currentcolor 0%, currentcolor 50%, Canvas 50%);
+  background-repeat: repeat-x;
+  background-size: calc(2 * var(--sbb-pearl-chain-spacing-small)) var(--sbb-pearl-chain-height);
+  inset-inline-end: var(--sbb-pearl-chain-height);
+
+  @include sbb.if-forced-colors {
+    background: unset;
+    border-block-start: #{sbb.px-to-rem-build(1)} dashed Highlight;
+    transform: translateY(#{sbb.px-to-rem-build(1)});
+  }
+}
+
+// Box-sizing rules contained in typography are not traversing Shadow DOM boundaries. We need to include box-sizing mixin in every component.
+@include sbb.box-sizing;
+
+:host {
+  --sbb-pearl-chain-height: var(--sbb-border-width-2x);
+  --sbb-pearl-chain-spacing-small: #{sbb.px-to-rem-build(2)};
+  --sbb-pearl-chain-color: var(--sbb-pearl-chain-bullet-color);
+  --sbb-pearl-chain-color-disruption: var(--sbb-pearl-chain-bullet-color-disruption);
+  --sbb-pearl-chain-color-past: var(--sbb-pearl-chain-bullet-color-past);
+  --sbb-pearl-chain-leg-width: 100%;
+  --sbb-pearl-chain-leg-animation: none;
+
+  display: contents;
+
+  @include sbb.pearl-chain-bullet-variables;
+}
+
+.sbb-pearl-chain__leg {
+  flex: var(--sbb-pearl-chain-leg-weight, 1) var(--sbb-pearl-chain-leg-weight, 1);
+  position: relative;
+  height: var(--sbb-pearl-chain-height);
+  border-inline-end: var(--sbb-pearl-chain-last-leg-margin, var(--sbb-pearl-chain-spacing-small))
+    solid Canvas;
+  background-color: currentcolor;
```

Comment:
Basic question: why do you define color and then define background with currentcolor? as there is no text at all in this component, couldn't we directly style the background?